src/FileSystem-Path/Path.class.st

```diff
@@ -339,6 +339,13 @@ Path >> do: aBlock [
 ]
 
 { #category : #accessing }
+Path >> emptyPathString [
+	"Answer the string representing an empty (size = 0) instance of the receiver"
+
+	^String with: self delimiter
```

Comment:
Why not just

```smalltalk
self delimiter asString
```

Otherwise this assumes that the delimiter is always a character...

Reply:
Good idea.  PR updated.

Thanks,
Alistair